test/fixtures/unicorn/prefer-event-key.fail.ts

```diff
@@ -0,0 +1,3 @@
+window.addEventListener('keydown', event => {
+	console.log(event.keyCode);
```

Comment:
Are these files formatted by Prettier? Doesn't look like so.

Reply:
they are purposely not formated by Prettier. This are just fixtures with a very specific focus. Sometimes the prettier format interferes with the fail fixtures.

Reply:
This was fixed in: https://github.com/ridedott/eslint-config/pull/319/commits/f05dbdcd336c36db4f7791a6ca333ef540ac77f9